themes/app/src/scss/_typography.scss

```diff
@@ -1,12 +1,160 @@
+$font-path: './webfonts';
+
+@font-face {
+    font-family: 'SourceSansPro';
+    font-style: normal;
+    font-weight: 300;
+    src: url('#{$font-path}/SourceSansPro-Light.woff') format('woff');
+}
+
+@font-face {
+    font-family: 'SourceSansPro';
+    font-style: normal;
+    font-weight: 400;
+    src: url('#{$font-path}/SourceSansPro-Regular.woff') format('woff');
+}
+
+@font-face {
+    font-family: 'SourceSansPro';
+    font-style: normal;
+    font-weight: 600;
+    src: url('#{$font-path}/SourceSansPro-SemiBold.woff') format('woff');
+}
+
 @font-face {
-    font-family: Federant;
+    font-family: 'SourceSansPro';
     font-style: normal;
-    font-weight: normal;
-    src: url('./webfonts/Federant-Regular.woff');
+    font-weight: 700;
+    src: url('#{$font-path}/SourceSansPro-Bold.woff') format('woff');
 }
 
-body {
-    font-family: Federant, serif;
+h1 {
+    font-family: 'SourceSansPro', sans-serif;
+    font-weight: 700;
+    font-size: 32px;
+    color: $dark-grey;
+    line-height: 1.2;
+    margin: 8px 0;
+}
+
+h2 {
+    font-family: 'SourceSansPro', sans-serif;
+    font-weight: 400;
+    font-size: 28px;
+    color: $navy-blue;
+    line-height: 1.2;
```

Comment:
Could be made into a line weight variable for easy changes in the future :)

Reply:
Not for styleguide, but will keep that in mind next time for others 😃